packages/clay-popover/src/index.tsx

```diff
@@ -247,10 +247,18 @@ const ClayPopover = React.forwardRef<HTMLDivElement, IProps>(
 				}
 			};
 
+			const onBlur = (event: FocusEvent) => {
+				if (event.view === undefined) {
+					setShow(false);
+				}
```

Comment:
It doesn't seem to work well in the case when you are browsing via keyboard and the focus moves to the popover and then when you press the tab when leaving the popover it still remains open, it only closes after the focus leaves the viewport. I think the best thing would be to have a reference to the popover and check if the `event.target` is inside the popover so we don't close the popover when there are other focusable elements, we do this in drop down and modal.

Reply:
Thanks @matuzalemsteles, I'll work on those improvements.